cookbooks/aws-parallelcluster-computefleet/templates/compute_fleet_status/is_fleet_ready.erb

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+
+sinfo_output=$(<%= node['cluster']['slurm']['install_dir'] %>/bin/sinfo -h -o '%N %t' | grep -v -E '(idle|alloc|mix|maint)$')
+while IFS= read -r line; do
+    nodelist=$(echo "$line" | awk '{print $1}')
+    <%= node['cluster']['slurm']['install_dir'] %>/bin/scontrol show hostnames "$nodelist" | { grep -E '^[a-z0-9\-]+\-st\-[a-z0-9\-]+\-[0-9]+.*' || true; }
```

Comment:
why do we want the hostname?

Reply:
`scontrol show hostnames` decode the slurm nodes shorthand representation to single nodes. For example, `scontrol show hostnames node-[1-3]` outputs
```
node-1
node-2
node-3
```